src/lib.rs

```diff
@@ -211,6 +211,11 @@ pub struct ErlNifPid {
     pid: ERL_NIF_TERM,
 }
 
+/// See [enif_make_pid](http://erlang.org/doc/man/erl_nif.html#enif_make_pid) in the Erlang docs
+pub fn enif_make_pid(_env: *mut ErlNifEnv, pid: & ErlNifPid) -> ERL_NIF_TERM {
```

Comment:
I had to wring my hands a bit on this one because `pid` is a reference here instead of a pointer, and the rest of erlang_nif-sys API is entirely pointer based.  But, I can't think of any situations where this will cause a problem so I will accept this as-is.